site/content/integrate/apps/call/_index.md

```diff
@@ -133,7 +133,6 @@ There are several types ([godoc](https://pkg.go.dev/github.com/mattermost/matter
 | :---------------- | :----- | :------------------ |
 | `type`            | string | Use `navigate`.     |
 | `navigate_to_url` | string | URL to navigate to. |
-| `navigate_to_url` | string | URL to navigate to. |
```

Comment:
Given that you already touched the code: Can we document https://github.com/mattermost/mattermost-plugin-apps/blob/ecf4e285e1a7fb2f85b0c7a2b5870f98a22a81fb/apps/call.go#L124-L124 here?

Reply:
done.  Please let me know if the wording is ok.  I also removed another duplicate.